nistats/tests/test_first_level_model.py

```diff
@@ -7,6 +7,12 @@
 
 import os
 import shutil
+
+try:
+    from tempfile import TemporaryDirectory
+except ImportError:
+    from nistats.utils import TemporaryDirectory
```

Comment:
In which case do you need your version of `TemporaryDirectory` ? A comment would be welcome here.